From a5060279e9f57e0fb6a3d66ff1512c954482c19e Mon Sep 17 00:00:00 2001
From: Ivan Shapovalov <intelfx@intelfx.name>
Date: Sun, 3 Jun 2018 00:14:45 +0300
Subject: [PATCH 7/9] Makefile: create manpages in subdirs via their Makefiles

Consistently use icc2ps/Makefile and osx-hotplug/Makefile
to generate respective makefiles. Otherwise subtle races can happen if
the same man page is generated both via the top-level Makefile and the
subdir Makefile.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ea75080..f4a806b 100644
--- a/Makefile
+++ b/Makefile
@@ -1514,7 +1514,7 @@ install-doc: doc
 
 GROFF=/usr/local/test/bin/groff
 GROFF=groff
-manual.pdf: $(MANPAGES) icc2ps/foo2zjs-icc2ps.1 osx-hotplug/osx-hplj-hotplug.1
+manual.pdf: man
 	-$(GROFF) -t -man \
 	    `ls $(MANPAGES) \
 		icc2ps/foo2zjs-icc2ps.1 \
-- 
2.19.0

